iothub/service/src/FeedbackRecord.cs

```diff
@@ -7,25 +7,51 @@ namespace Microsoft.Azure.Devices
     using Newtonsoft.Json;
 
     /// <summary>
-    /// Contains properties of an individual feedback record. 
+    /// Contains properties of an individual feedback record.
     /// </summary>
     public class FeedbackRecord
     {
+        /// <summary>
+        /// The MessageId of the cloud-to-device message to which this feedback information relates.
```

Comment:
I think when describing something generically, you wouldn't treat it like a variable (e.g. `MessageId`). If it were just plain English, I'd say `The Id of the message of the...`

Reply:
Or if that is too wordy, you could do `The message Id of the...`

Reply:
sure sounds good